src/interpreter/runtime/values/f32.js

```diff
@@ -42,6 +42,30 @@ export class f32 extends Float<i32> {
 }
 
 export class f32nan extends f32 {
+  reinterpret(): i32 {
```

Comment:
it would be useful if you could add a comment which references the relevant section in the spec that defines this behaviour - thanks!

Reply:
That's a good point, but I could not find much in the spec about it. The conversion directly reflects the  IEEE 754-2008 standard for `nan`s, but I linked it anyways. Lmk if you think this is helpful or more confusing.